### DISCUSSION TITLE
Benefits of having dedicated RSE masters: Intention: convince others.

### DISCUSSION THREAD

**Discussion by CaptainSifff** [the-teachingRSE-project/RSE-Masters#35] [General]
Benefits of having a dedicated RSE masters course/curriculum: Intention: convince others.

**Comment by jpthiele** [comment]
While integration of RSE topics into existing domain classes and (new) specific RSE classes are certainly a good way to train existing staff and domain scientists, it does not fully cover the need for staff in central RSE groups.
Especially when it comes to experience with interdisciplinary projects. 

Therefore, we see a dedicated RSE master as a good way to obtain a dedicated workforce for RSE positions. 
Additionally, this establishes the role of an RSE as a viable and clear career path.

**Comment by CaptainSifff** [comment]
@jcohen02 expressed interest.

**Comment by jcohen02** [comment]
**Benefits of having dedicated RSE masters programmes**

Some motivation to support this discussion:

 - There is a significant skills shortage emerging in the RSE space (growing demand for software, data, computing infrastructure skills).
 - RSE skills are now required across almost all research domains, a Masters programme is a great way of providing the core skills to researchers with undergraduate training in another domain who now need to develop these skills.
 - You can learn to develop software without specialist training but this often results in software that doesn't support/offer sustainability, robustness, reproducibility?!
 - There are many skills to learn to become an experienced RSE, combining these into a single qualificiation is a good way to deliver a set of advanced training that may otherwise need to be gained over several years of work experience.

Some possible areas for discussion:

 - Who would such programmes be targeted at?
 - Where would the people who graduate from such a scheme go in terms of future work? [_@annalenalamprecht_]
 - Do we need a full Masters programme or can people gain the skills through other more targeted means? [_@ljgarcia_]
 - Would such a programme be designed to support people looking for a research/academic career, an RSE-focused career or both? (comment above from @jpthiele suggests this could focus on the latter).
 - How many students would need to be trained to make a Masters programme viable?
 - What are the core RSE-related topics that discussion participants see are lacking within their domains that such a Master programme would provide?

**Comment by jdobber** [comment]
> Some possible areas for discussion:

* Why not pursue a masters degree in computer science for non computer scientists? Examples: [Informatik für Geistes- und Sozialwissenschaftler](https://www.tu-chemnitz.de/informatik/studium/studiengaenge/ma_informatik_fuer_geistes_und_sozialwissenschaftler.php#aufbau), [Computer Science @ York](https://online.york.ac.uk/msc-computer-science/)
* Why not pursue another bachelor degree in computer science (or data science, business informatics...)?
* Why not pursue a research-oriented masters degree in my home domain with some taught modules about software engineering?
